README.md

```diff
@@ -16,12 +16,12 @@ Users can create their own test hierarchies, create test factories for generatin
 
 ## Documentation
 
-The official documentation is maintained [here](https://eth-cscs.github.io/reframe/index.html).
+The official documentation is maintained [here](https://reframe-hpc.readthedocs.org).
```

Comment:
If the redirection from the old page succeeds, we don't need to change the README here.